Fix the treble violations introduced by crashlogd

ceiba1985 opened the pull request:
Vendor data files should be put under /data/vendor basing on Google's requirement, and their file contexts mustn't be core_data_file_type.

Tracked-On: OAM-117257

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

ceiba1985 committed:
Fix the treble violations introduced by crashlogd

Vendor data files should be put under /data/vendor basing on Google's
requirement, and their file contexts mustn't be core_data_file_type.

Tracked-On: OAM-117257
Signed-off-by: jizhenlo <zhenlong.z.ji@intel.com>

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /cactus-absp-or-local/celadon_smr0_apollo-merge/114
